proposals/lip-0046.md

```diff
@@ -268,9 +272,8 @@ Let `blockAssetBytes` be the bytes included in the block asset for the Random mo
 
 After all the transactions of a block `b` are executed, the following logic is applied:
 
-* While the size of the `validatorReveals` array is larger than `MAX_LENGTH_VALIDATOR_REVEALS`, delete the element of `validatorReveals` array with the smallest value of `height`.
+* If the size of the `validatorReveals` array is equal to `MAX_LENGTH_VALIDATOR_REVEALS`, delete the element of `validatorReveals` array with the smallest value of `height`.
```

Comment:
Comment about line 264: I think it is good to add at some place a link to LIP55 definition of assets, since there is not any reference in this LIP; since block assets are not used in so many LIPs, I think it would be good to make easier for the reader to navigate.

Reply:
I already referenced the link about block assets in the table.